src/app/legacy/containers/Header/index.jsx

```diff
@@ -46,17 +45,11 @@ const HeaderContainer = ({
   scriptSwitchId = '',
   renderScriptSwitch = true,
 }) => {
-  const { pageType, isAmp, isApp } = useContext(RequestContext);
+  const { isAmp, isApp, pageType } = useContext(RequestContext);
   const { service, script, translations, dir, scriptLink, lang, serviceLang } =
     useContext(ServiceContext);
   const { skipLinkText } = translations;
 
-  // The article page toggles the nav bar based on environment
-  const showNavOnArticles = useToggle('navOnArticles').enabled;
-
-  // All other page types show the nav bar at all times
-  const showNav = showNavOnArticles || pageType !== ARTICLE_PAGE;
```

Comment:
Just getting rid of this as it seems to be really old config from trialling Article pages.